### DISCUSSION TITLE
no privileges to modify /proc/sys/user/max_user_namespaces

### DISCUSSION THREAD

**Discussion by scott-5** [containers/podman#16769] [Q&A]
I wanna use podman in no-root user. I have no root privileges, so although I have installed podman but can not modify /proc/sys/user/max_user_namespaces from 0. Does this mean that as long as you have not any root privileges, you must not run podman? Can I specify the directory of max_user_namespaces in my user account?

**Comment by rhatdan** [comment]
That will block you being able to create a user namespace, as I understand it.  Bottom line no rootless podman without user namespace.

You could confirm this by playing with the unshare command or executing a podman unshare

**Reply by scott-5** [reply]
Thanks a lot. Could you tell me more about how to deal with having no privileges to modify /proc/sys/user/max_user_namespace? To build a podman unshare or whether it can be run without user namespace? My podman installed by conda `conda install -c conda-forge podman` in my $HOME directory.

**Reply by rhatdan** [reply]
Basically the OS that you are installing on thought that the kernels version of User Namespace was not mature enough to allow non privileged users to use user namespace.  Therefore administrators need to set a flag, saying they are willing to accept the risk. You need to get an Admin of your system to accept the risk or move to a newer OS.

**Reply by scott-5** [reply]
Okey. I have understood.
